### PR TITLE
fix #73781 escapegames24.com

### LINKED ISSUE #73781: escapegames24.com
### Issue URL (Anti Adblock Script)
 [http://www.escapegames24.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.escapegames24.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 13
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#73781

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden There are ads' placeholders. I couldn't fix all, so can you take a look?

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -544,6 +544,8 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/73781
+escapegames24.com#%#//scriptlet("prevent-addEventListener", "load")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:548]: It's too wide. Please exchange to the `escapegames24.com#%#//scriptlet("prevent-setTimeout", "ads", "2000")`.
Also add the rule below to the `content_blocker.txt` (below common rules like `blockadblock` and `g207` sections).
```
! https://github.com/AdguardTeam/AdguardFilters/issues/73781
@@||escapegames24.com^$generichide
```

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
> There are ads' placeholders. I couldn't fix all, so can you take a look?

What kind of placeholders? Try these rules
```
escapegames24.com#?#.sidebar > div.widget:has(> h2:contains(Advertisement))
escapegames24.com#?#.widget-content > div[align="center"]:has(> font > b:contains(Advertisement))
```

**Comment by wout-codes** [comment]
> > There are ads' placeholders. I couldn't fix all, so can you take a look?
> 
> What kind of placeholders? Try these rules
> 
> ```
> escapegames24.com#?#.sidebar > div.widget:has(> h2:contains(Advertisement))
> escapegames24.com#?#.widget-content > div[align="center"]:has(> font > b:contains(Advertisement))
> ```

Where should I add this one?

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt]: Should I add at that two places? Or just one?

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt]: What do you mean by wide rule even?

**Comment by zloyden** [comment]
@wout-codes don't make commits right now, I'll show you.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Ok, done. You can see where it should be.

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt]: It will prevent all `eventListener` with `load`. Without third parameter it's wide.

**Pull request merged**